src/ragas/metrics/base.py

```diff
@@ -156,3 +157,30 @@ def init(self, run_config: RunConfig):
                 f"Metric '{self.name}' has no valid embeddings provided (self.embeddings is None). Please initantiate a the metric with an embeddings to run."  # noqa
             )
         self.embeddings.set_run_config(run_config)
+
+
+class Ensember:
+    def from_discrete(self, inputs: list[list[t.Dict]], attribute: str):
+        assert all(
+            len(item) == len(inputs[0]) for item in inputs
+        ), "all inputs must have the same length"
+
+        assert all(
+            attribute in item for input in inputs for item in input
+        ), "attribute not found in all items"
+
+        if len(inputs) == 1:
+            return inputs[0]
+
+        verdict_agg = []
+        for i in range(len(inputs[0])):
+            item = inputs[0][i]
+            verdicts = [inputs[k][i][attribute] for k in range(len(inputs))]
+            verdict_counts = dict(Counter(verdicts).most_common())
+            item[attribute] = list(verdict_counts.keys())[0]
+            verdict_agg.append(item)
+
+        return verdict_agg
```

Comment:
how does this work?

Reply:
I just ensembles multiple outputs. 
For example if results is [0,1,1] over 3 llm calls for the query, output will be 1 which is the majority vote. I will add a doc for this.